src/components/calendar/CalendarControls.tsx

```
@@ -0,0 +1,97 @@
+      year: 'numeric',
+    });
+  }
+
+  return (
+    <div class="flex w-full flex-wrap items-center justify-between gap-4">
+      <div class="text-xl font-bold">{extractLabel()}</div>
+      <div class="flex items-center gap-2">
+        <button
+          onClick={() => setDisplayDate(new Date())}
+          class="hidden rounded-md bg-navy-800 px-3 py-1.5 text-sm font-medium text-white hover:bg-navy-700 md:block"
+        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)